h5p-bildetema/src/components/Language/Language.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styles from "./Language.module.scss";
+import { Language as LanguageType } from "../../../../common/types/types";
+
+type SelectProps = {
+  language: LanguageType;
+};
+
+export const Language: React.FC<SelectProps> = ({
+  language: { label, code },
+}) => {
+  return (
+    <div className={styles.language}>
+      <div>
+        <label htmlFor={code} className={styles.input_container}>
+          <input type="checkbox" id={code} />
```

Comment:
The `htmlFor` attribute is not needed when the label wraps the input (and no other inputs).

```suggestion
        <label className={styles.input_container}>
          <input type="checkbox" />
```

Reply:
![image](https://user-images.githubusercontent.com/71496242/170948141-95457afd-b97a-4c23-8ced-358d96a565e4.png)
Got some eslint error when i didnt have it